saleor/dashboard/product/forms.py

```diff
@@ -28,6 +28,13 @@ class StockForm(forms.ModelForm):
     class Meta:
         model = Stock
         exclude = ['quantity_allocated', 'variant']
+        labels = {
+            'location': pgettext_lazy(
+                'Stock location', 'Location'),
+            'quantity': pgettext_lazy(
+                'Stock quantity', 'Quantity'),
```

Comment:
Do we want this to be translated differently than other "quantity" labels? If not, you can use the same context here.